sentry_streams/sentry_streams/pipeline/pipeline.py

```diff
@@ -91,26 +100,17 @@ def merge(self, other: Pipeline, merge_point: str) -> None:
         }
 
         for step in other.steps.values():
-            if step.name not in other_pipeline_sources:
-                self.register(step)
+            self.register(step)
```

Comment:
Please correct the docstring of the `merge` method. It still reference the `context` field of the steps that have been removed.